pavelib/quality.py

```diff
@@ -298,6 +298,50 @@ def run_jshint(options):
         )
 
 
+@task
+@needs('pavelib.prereqs.install_python_prereqs')
+@cmdopts([
+    ("limit=", "l", "limit for number of acceptable violations"),
+])
+def run_safelint(options):
+    """
+    Runs safe_template_linter.py on the codebase
+    """
+
+    violations_limit = int(getattr(options, 'limit', -1))
+
+    safelint_report_dir = (Env.REPORT_DIR / "safelint")
+    safelint_report = safelint_report_dir / "safelint.report"
+    _prepare_report_dir(safelint_report_dir)
+
+    sh(
+        "/edx/app/edxapp/edx-platform/scripts/safe_template_linter.py >> {safelint_report}".format(
```

Comment:
@jzoldak Any recommendation for making this a recognizable command, rather than providing the full path?  This was just temporary for getting this started.